appveyor.yml

```diff
@@ -0,0 +1,17 @@
+version: 1.0.{build}
+init:
+- git config --global core.autocrlf true
+install:
+- cd src
+- nuget restore
+- cd ..
+build_script:
+- cd build
+- mkdir c:\projects\keys
+- '"C:\Program Files (x86)\Microsoft SDKs\Windows\v8.1A\bin\NETFX 4.5.1 Tools\sn.exe" -k c:\projects\keys\openstacknetsdk.snk'
```

Comment:
I believe the solution uses keys in a different location now, so we'll need to either add additional lines here, or merge a pull request which adds the keys to source control and remove the line altogether.

I'm working on getting the list of keys now in use.